diff -up bindgraph-0.3/bindgraph.pl.1~ bindgraph-0.3/bindgraph.pl
--- bindgraph-0.3/bindgraph.pl.1~	2024-01-01 23:10:55.242626793 +0100
+++ bindgraph-0.3/bindgraph.pl	2024-01-01 23:24:01.257407923 +0100
@@ -8,7 +8,7 @@
 
 my $rrdstep = 60;
 
-my $daemon_logfile = 'bindgraph.log';
+my $daemon_logfile = '/var/log/bindgraph/bindgraph.log';
 my $daemon_rrd_dir = '.';
 my $daemon_pidfile;
 my $rrd = 'bindgraph.rrd';
